### ISSUE TITLE
problems about cruiseMLP prediction model 

### ISSUE THREAD

**Issue by darlingwang2** [ApolloAuto/apollo#14008]
My problems are about cruiseMLP prediction model, model outputs are three features---labels saved for each lane-sequence, and finish time.
Offline training generates the ground-truth and puts the data into model training, without distinguishing go or cut-in lane-sequence. But online model inference use two models---go and cut-in model. And it uses the label for lane-sequence to represent probability, it means the label is in proportional to probability, whereas the label -1:false cut-in  1,3:true cut-in  0:false go  2,4:true go .
As for the solution, why not separate go and cut-in scenario during offline training? In details, according to the label ,we can separate the training set into go and cut-in sets, and verify the label into 1 or 0, showing whether to go or cut-in.
